CONTRIBUTING.md

```diff
@@ -49,6 +55,33 @@ All changes must be code reviewed. Coding conventions and standards are explaine
 docs](https://github.com/kubernetes/community/tree/master/contributors/devel). Expect reviewers to request that you
 avoid common [go style mistakes](https://github.com/golang/go/wiki/CodeReviewComments) in your PRs.
 
+## Reviewing a Patch
```

Comment:
should this link to `REVIEWING.md`?

Reply:
nevermind just saw https://github.com/kubernetes-sigs/cluster-api/pull/3191/files#r440457368